### ISSUE TITLE
XML report includes semicolon after PLUGIN_ERROR id and message

### ISSUE THREAD

**Issue by mpaluchowski** [cflint/CFLint#241]
When scanning the following `test.cfm` file:

```coldfusion
<cfscript>
  defaultFontColor = "##444444";
</cfscript>
```

with `java -jar CFLint-1.0.0-all.jar -file test.cfm -xml`, the XML report contains:

```xml
<?xml version="1.0" encoding="UTF-8" ?>
<issues version="1.0.0">
  <issue severity="ERROR" id="PLUGIN_ERROR:" message="PLUGIN_ERROR:" category="CFLint" abbrev="PE">
    <location file="test.cfm" fileName="test.cfm" function="" column="1" line="1" message="" variable="">
      <Expression><![CDATA[]]></Expression>
    </location>
  </issue>
  <issue severity="ERROR" id="PLUGIN_ERROR" message="PLUGIN_ERROR" category="CFLint" abbrev="PE">
    <location file="test.cfm" fileName="test.cfm" function="" column="22" line="2" message="Error in plugin: class java.lang.NullPointerException" variable="class java.lang.NullPointerException">
      <Expression><![CDATA[]]></Expression>
    </location>
  </issue>
</issues>
```

The first issue incorrectly contains a semicolon `:` after `PLUGIN_ERROR` in `id` and `message` attributes. The second one is fine. If there are more plugin errors from the same file, then most of them have the erroneous semicolon.

Of course, these errors shouldn't be triggered for the file in the first place, but that's already reported in #240.

**Comment by ryaneberly** [comment]
fixed.  Thanks!
